javascript/common/common.js

```diff
@@ -0,0 +1,202 @@
+        location: "Banglore",
+        description: "Configure the providers that are available to users when they sign in"
+    }
+];
+let employee;
+let roleDetails;
+
+employee = JSON.parse(localStorage.getItem('employee'))?.length ? JSON.parse(localStorage.getItem('employee')) : localStorage.setItem('employee', JSON.stringify(employeeMockData));
+roleDetails = JSON.parse(localStorage.getItem('roleDetails'))?.length ? JSON.parse(localStorage.getItem('roleDetails')) : localStorage.setItem('roleDetails', JSON.stringify(roleMockDetails));
+
+
+
+window.document.addEventListener("DOMContentLoaded", function () {
+    var today = new Date(Date.now() - 7 * 24 * 60 * 60 * 1000).toISOString().split('T')[0];
+    document.getElementsByName("todaysdate")[0]?.setAttribute('min', today);
+    let headerHeight = document.querySelector(".header").offsetHeight;
+    let descriptionHeight = document.querySelector(".employee-section").offsetHeight;
+    let filterHeight = document.querySelector(".filter-section").offsetHeight;
+    let advanceFilterHeight = document.querySelector(".advance-filter-section").offsetHeight;
+    let deleteSectionHeight = document.querySelector(".change-section").offsetHeight;
+    let totalHeight = headerHeight + descriptionHeight + filterHeight + advanceFilterHeight + deleteSectionHeight;
+    totalHeight += 50;
+    let tableHeight = "calc(100vh - " + totalHeight + "px)";
+    document.querySelector(".table-div").style.height = tableHeight;
+});
+
+// toggle side Menu
+var sideBarOpen = true;
+var mobileViewSideBarOpen = false;
+
+
+const sideBar = document.querySelector('#side-nav');
+const notification = document.querySelector(".pop-up");
+const handleIcon = document.getElementById("handle-logo");
+const mainContent = document.querySelector(".main-content");
+const minSideBar = document.querySelector(".minimized-side-nav");
+
+var screenSize = window.matchMedia("(min-width: 720px)");
+let toggleSideBar = () => {
+    const mainContentWidth = sideBar.offsetWidth + 15;
+    if (screenSize.matches) {
+        if (sideBarOpen) {
+            closeSideBar(mainContentWidth);
+        }
+        else {
+            openSideBar(mainContentWidth);
+
+        }
+        sideBarOpen = !sideBarOpen;
+    }
+    else {
+        if (mobileViewSideBarOpen) {
+            mobileViewcloseSideBar();
+        }
+        else {
+            mobileViewopenSideBar();
+
+        }
+        mobileViewSideBarOpen = !mobileViewSideBarOpen;
+    }
+
+}
+
+function mobileViewopenSideBar() {
+    sideBar.style.display = "flex";
+    sideBar.style.position = "absolute";
+    sideBar.style.width = "50%";
+    sideBar.style.zIndex = 11;
+    handleIcon.style.marginLeft = "7.5rem";
+    handleIcon.style.zIndex = 12;
+    handleIcon.style.transform = "rotate(360deg)";
+    handleIcon.style.marginTop = "0rem";
+}
+
+function mobileViewcloseSideBar() {
+    sideBar.style.display = "none";
+    handleIcon.style.marginLeft = "-2rem";
```

Comment:
Why are you adding styles here?
anyhow you are getting these elements in global scoped, can we add respective classes to the elements.